app/models/stripe_wrapper.rb

```diff
@@ -0,0 +1,46 @@
+module StripeWrapper
+
+  DEFAULT_CURRENCY = "usd"
+
+  class Charge
+    def initialize(status, responses = {})
+      @status = status
+      @customer = responses[:customer]
+      @charge = responses[:charge]
+      @error = responses[:error]
+    end
+
+    def self.create(options = {})
+      begin
+        customer = Stripe::Customer.create(
+          source: options[:token],
+          description: options[:customer]
+        )
```

Comment:
The wrong part of this is that, you are in class Charge. So you should create another class for customer to wrap the behaviour there.

The other thing is that when you would like to charge again - you would create the new customer for every charge. Its better to check whether customer exists and if not then create him. If the customer exists - then retrieve him and use his record.